elastio-velero-integration/README.md

```diff
@@ -0,0 +1,94 @@
+This article provides a comprehensive guide on the backup and restore procedure for snapshots of EKS persistent storage created by Velero. 
+Elastio, a powerful tool, efficiently reduces storage costs on AWS by effectively backing up and compressing data.
+
+## Requirements
+- AWS CLI
+- Elastio CLI
+- jq tool
+
+## Prerequisites
+Before getting started, ensure that you have a Linux instance with Elastio and AWS CLI connected to your AWS account with Velero backups configured. Additionally, make sure to install the `jq` tool.
+
+## Backup procedure
+
+### Backup via Elastio Tenant
+To create a backup policy that imports all snapshots corresponding to desired EBS volumes, you can follow these steps:
+
+1. Access your Elastio Tenant.
+2. Navigate to the Policies section.
+3. Click on "New Policy" to start creating a new policy.
+4. Provide a meaningful name for the policy.
+5. Specify the appropriate schedule for the backups (e.g., daily, weekly, etc.) based on your requirements.
+6. Select Integrity Scan options if required.
+7. Add the desired EBS volumes that you want to include in the backups.
+8. Configure additional options for the policy like on the screenshot below:
+![image](https://github.com/elastio/contrib/assets/81738703/3f3e0103-806e-435e-870c-79b98caf5748)
+9. Save the policy configuration.
+   
+Once the policy is created, Elastio will automatically import all the snapshots corresponding to the specified EBS volumes according to the defined schedule.
+
+### Backup via Elastio CLI
+To backup all snapshots associated with the EBS volume, simply execute the following command. Optionally you can include the `--iscan` parameter to enable vulnerability scanning for the backups.
+
+```
+elastio ebs import-snapshot --volume-id [Volume-id] --iscan
+```
+
+## TODO: Add lambda desctiption for snapshots removal
```

Comment:
@Veetaha has a separate PR in `contrib` (#48) that goes over this in detail.  It's probably sufficient here just to link to that once it lands.

Reply:
Added reference to this article.